src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Build.ps1

```diff
@@ -984,10 +986,10 @@ class Build: ADOSVTBase
                     }
                 }
 
-                if(($editableVarGrps | Measure-Object).Count -gt 0)
+                if($editableVarGrps.Count -gt 0)
                 {
-                    $controlResult.AddMessage("Total number of variable groups on which contributors have edit permissions in build definition: ", ($editableVarGrps | Measure-Object).Count);
-                    $controlResult.AdditionalInfo += "Total number of variable groups on which contributors have edit permissions in build definition: " + ($editableVarGrps | Measure-Object).Count;
+                    $controlResult.AddMessage("Total number of variable groups on which contributors have edit permissions in build definition: $($editableVarGrps.Count)");
+                    $controlResult.AdditionalInfo += "Total number of variable groups on which contributors have edit permissions in build definition: " + $editableVarGrps.Count;
```

Comment:
Code is using $editableVarGrps.Count a multiple location.
can we store in a separate variable before if block and then use it further